src/sentry/conf/server.py

```diff
@@ -753,12 +753,12 @@ def create_partitioned_queues(name):
     'organizations:sso-saml2': False,
     'organizations:sso-rippling': False,
     'organizations:group-unmerge': False,
-    'organizations:github-apps': True,
-    'organizations:integrations-v3': True,
+    'organizations:github-apps': False,
```

Comment:
Did you mean to commit these changes?

Reply:
Oh, okay. Weird, not sure how I ended up seeing them. Carry on!

Reply:
yeah sorry about that!